src/test/java/org/sep3tools/SepExamplesTest.java

```diff
@@ -82,7 +82,7 @@ public void verifyFremdDatenfeldExamples() {
 	public void verifySepExamples(String df, String propFile) {
 		JavaConnector.setPropertiesFile(DBPROPFILENAME);
 		JavaConnector.setDf(df);
-		Properties properties = loadPropertiesFromFile(propFile, "=");
+		Properties properties = loadPropertiesFromFile(propFile, " = ");
```

Comment:
@wanhoff Was ist der Hintergrund für diese Änderung?

Reply:
Hintergrund ist, dass '=' nun in den auszuwertenden Zeichenketten vorkommen kann. Es wird bei Farben als 'lich' übersetzt. Daher kann es nicht mehr als einzelnes Zeichen in .properties-Dateien verwendet werden (Ein Doppelpunkt kann auch in den Zeichenketten vorkommen). Daher trenne ich für die Tests nun mit einem in Leerzeichen eingerahmten '='.